.circleci/config.yml

```diff
@@ -101,6 +105,21 @@ jobs:
       browser: << parameters.browser >>
       bver: << parameters.bver >>
     steps: [run-tests]
+  ExecuteReleaseDummy:
+    executor: docker-with-browser
+    steps:
+      - get-code-and-dependencies
+      - lint
+      - unit-tests
+      - run: echo Wait for other jobs in the workflow to finish
+      - run: echo Release will execute "<< pipeline.parameters.releaseCommand >>"
+  ExecuteRelease:
+    executor: docker-with-browser
+    steps:
+      - get-code-and-dependencies
+      - lint
+      - unit-tests
+      - run: echo <<pipeline.parameters.releaseCommand>>
```

Comment:
Will this run the command or just echo it?

Reply:
it will just echo it.

Reply:
thanks for catching this. I will fix to really execute the command.